src/main/java/org/datadog/jenkins/plugins/datadog/traces/IdGenerator.java

```diff
@@ -0,0 +1,13 @@
+package org.datadog.jenkins.plugins.datadog.traces;
+
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Keeps the ID generation logic for the TraceSpan
+ */
+public class IdGenerator {
+
+    public static long generate(){
+        return ThreadLocalRandom.current().nextLong(1, Long.MAX_VALUE);
```

Comment:
is this copy paste from the tracer? will it be random enough?

Reply:
Yes, it's the same approach that is used in the Java Tracer: https://github.com/DataDog/dd-trace-java/blob/master/dd-trace-api/src/main/java/datadog/trace/api/IdGenerationStrategy.java#L10